packages/backend/src/projectAdapters/dbtGitProjectAdapter.ts

```diff
@@ -40,7 +41,12 @@ const stripTokensFromUrls = (raw: string) => {
     return raw.replace(pattern, '//*****@');
 };
 
-const gitErrorHandler = (e: Error, repository: string) => {
+const gitErrorHandler = (e: unknown, repository: string) => {
+    if (!(e instanceof Error)) {
```

Comment:
passing unknown to this method, and checking if type is not error (should never happen)